index.html

```diff
@@ -5154,7 +5154,56 @@ <h2>The <code>application/credential+ld+json</code> Media Type</h2>
   </p>
 
 </section>
+<section id="verifiable-credential-media-type">
+  <h2>The <code>application/verifiable-credential+ld+json</code> Media Type</h2>
+  <p>
+    This specification registers the <code>application/verifiable-credential+ld+json</code> MIME Media Type specifically for
+    identifying documents
+    conforming to the Verifiable Credentials format.
+  </p>
+  <table>
+    <tr>
+      <td>Type name: </td>
+      <td>application</td>
+    </tr>
+    <tr>
+      <td>Subtype name: </td>
+      <td>verifiable-credential+ld+json</td>
+    </tr>
+    <tr>
+      <td>Required parameters: </td>
+      <td>None</td>
+    </tr>
+    <tr>
+      <td>Encoding considerations: </td>
+      <td>
+        Resources that use the "<code>application/verifiable-credential+ld+json</code>"
+        Media Type are required to conform to all of the requirements
+        for the "<code>application/credential+ld+json</code>" Media Type and are
```

Comment:
Points back at `application/credential+ld+json`... related to the debate that is not resolved on https://github.com/w3c/vc-data-model/pull/1014

Reply:
This makes me think (again) that 
*IF* we're going to have media types for both verifiable and unverifiable credentials
`application/verifiable-credential+ld+json` 
(hyphen, i.e., parallel type to `application/credential+ld+json`) 
should instead be 
`application/verifiable+credential+ld+json` 
(plus, i.e., subtype of `application/credential+ld+json`)